sdrf_pipelines/maxquant/maxquant.py

```
@@ -2463,10 +2465,10 @@ def parse_mods(mods):
     def maxquant_experiamental_design(self, sdrf_file, output):
         sdrf = pd.read_csv(sdrf_file, sep="\t")
         sdrf = sdrf.astype(str)
-        sdrf.columns = map(str.lower, sdrf.columns)
-        f = open(output, "w")
-        f.write("Name\tFraction\tExperiment\tPTM")
-        for index, row in sdrf.iterrows():
+        sdrf.columns = sdrf.columns.str.lower()
+        f = open(output, "w", encoding="utf-8")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)